src/composables/deatailsApi.js

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+const api = axios.create({
+    baseURL: process.env.VUE_APP_BASE_URL,
+});
+api.interceptors.request.use(
+    (config) => {
+        const token = JSON.parse(localStorage.getItem("token"));
+        if (token) {
+            config.headers.token = `${token}`;
```

Comment:
@ShivanshSinghnnovative any reason for assigning token like `${token}` ?
why not
config.headers.token = token;